enos/modules/softhsm_distribute_vault_keys/main.tf

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+terraform {
+  required_providers {
+    enos = {
+      source = "app.terraform.io/hashicorp-qti/enos"
```

Comment:
We should add a version requirement for >= 0.4.8 due to having enos_user in this module.